test/test.py

```diff
@@ -251,6 +268,15 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
         # noinspection PyDeprecation
         self.assertEqual(project.project_short_name, project.project_shortname)
 
+        if insdc_project_accessions != set() or project.insdc_project_accessions != set():
```

Comment:
I thought I had left a comment here in my previous review, but GH must have swallowed it.

The only way `insdc_project_accessions != set() or project.insdc_project_accessions != set()` can be false is if `insdc_project_accessions == set() and project.insdc_project_accessions == set()` but in that case the assertion would pass so there is no point in making it conditional.

Reply:
Good catch. The condition started out only checking `insdc_project_accessions != set()` but when `project.insdc_project_accessions != set()` was added the condition became superfluous. Will be corrected in next commit.